README.md

```diff
@@ -4,7 +4,7 @@
 
 [![Frogbot](images/frogbot-intro.png)](#readme)
 
-[![Scanned by Frogbot](https://raw.github.com/jfrog/frogbot/master/images/frogbot-badge.svg)](https://github.com/jfrog/frogbot#readme)
+[![Scanned by Frogbot](https://raw.github.com/jfrog/frogbot/master/images/frogbot-badge.svg)](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)
```

Comment:
Since this badge is also included in the [Frogbot Documentation](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot/frogbot-badge), let's also create a PR to update it there.